openedx_filters/learning/tests/test_filters.py

```diff
@@ -728,3 +729,26 @@ def test_course_run_api_render_started(self):
         result = CourseRunAPIRenderStarted.run_filter(serialized_courserun)
 
         self.assertEqual(serialized_courserun, result)
+
+
+class TestIDVFilters(TestCase):
+    """
+    Test class to verify standard behavior of the ID verification filters.
+    You'll find test suites for:
+
+    - IDVPageURLRequested
+    """
+
+    def test_course_idv_page_url_requested(self):
```

Comment:
nit: is there a reason that the test case includes course in it? Wondering if it should be:
```suggestion
    def test_idv_page_url_requested(self):
```

Reply:
Ah, this is another artifact from me copy/pasting tests.